aip/general/0008.md

```diff
@@ -114,18 +114,33 @@ followed by a bulleted list explaining the example.
 Individual subsections can be cited individually, and further elaborate
 details.
 
+## Rationale
+
+The "rationale" section is optional, and helps the reader understand the
+motivation for specific guidance within the AIP.
+
+Deeper explanations of design justification and tradeoffs **must** be in the
+rationale instead of other sections, to ensure the rest of the document acts as
+an easily digestible reference.
+
+## History
+
+The "history" section is optional, and documents events and context around
+a significant edit (e.g. rewrite) of an AIP.
+
+While the changelog is a dotted list of one-line summaries of every change to
+an AIP, the history section should elaborate on significant events in a
+freeform format.
+
+- The section **must not** be used to exhaustively enumerate all changes (this
+  is what the changelog provides).
```

Comment:
This is not a good use of a parenthetical and is a runon sentence as such. Break into two sentences.

> enumerate all changes... This is what the ...

Reply:
done.